### PR TITLE
wp: add download and config examples

### PR DESCRIPTION
**Pull request by iSnackyCracky**
Add commands for installing a fresh WordPress System

- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/wp.md [hunk 1]
```diff
@@ -11,6 +11,14 @@
 
 `wp cli update`
 
+- Download a fresh WordPress installation to current directory:
+
+`wp core download [--locale=<locale>]`
```
Comment (marchersimon) [inline pages/common/wp.md:16]: We don't really have syntax for optional arguments, but we can mention it in the description. Also, we use token syntax for everything which the user has to input:
```suggestion
- Download a fresh WordPress installation to current directory, optionally specifying the locale:

`wp core download --locale={{locale}}`
```

File: pages/common/wp.md [hunk 2]
```diff
@@ -11,6 +11,14 @@
...
+
+- Create basic wpconfig file:
```
Comment (marchersimon) [inline pages/common/wp.md:18]: We use backticks for technical terms.
```suggestion
- Create basic `wpconfig` file:
```

File: pages/common/wp.md [hunk 3]
```diff
@@ -11,6 +11,14 @@
...
+
+`wp config create --dbname=<dbname> --dbuser=<dbuser> [--dbpass=<dbpass>] [--dbhost=<dbhost>]`
```
Comment (marchersimon) [inline pages/common/wp.md:20]: As already said, we can't use the `[ ]` syntax from man pages. Are those last two flags useful? Either we write them without it or we remove them if they're not that important.

File: pages/common/wp.md (hunk 1 above)
Comment (iSnackyCracky) [reply pages/common/wp.md:16]: Ah, okay. Yeah that works fine for me 👍

File: pages/common/wp.md (hunk 3 above)
Comment (iSnackyCracky) [reply pages/common/wp.md:20]: I'd guess the --dbpass is important enough to specify it, --dbhost however probably isn't.

Comment (navarroaxel) [reply pages/common/wp.md]: These are very common DB arguments, I think we can left all.

**Review by marchersimon — APPROVED** [review]
Thanks @iSnackyCracky

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**